src/data/markdown/docs/40 xk6-disruptor/04 Examples/02 Inject HTTP faults into Pod.md

```diff
@@ -306,16 +307,15 @@ export const options = {
             preAllocatedVUs: 10,
             maxVUs: 100,
             exec: "default",
-            startTime: '0s',
-            duration: "30s",
+            startTime: "10s",
+            duration: "20s",
         },
         disrupt: {
             executor: 'shared-iterations',
             iterations: 1,
             vus: 1,
             exec: "disrupt",
-            startTime: "0s",
```

Comment:
Even when it is not necessary, I would prefer to leave this to make it more evident.